api-server/README.md

```diff
@@ -470,3 +465,18 @@ Please guarantee that the port is not occupied by other processes. If the port s
 ```
 
 If the Web UI is ready, you can navigate to `http://127.0.0.1:8080` to open the chatbot, it will interact with the API of your server.
+
+## Set Log Level
+
+You can set the log level of the API server by setting the `LLAMA_LOG` environment variable. For example, to set the log level to `debug`, you can run the following command:
+
+```bash
+wasmedge --dir .:. --env LLAMA_LOG=debug \
+    --nn-preload default:GGML:AUTO:llama-2-7b-chat.Q5_K_M.gguf \
```

Comment:
I think llama2 is outdated already, can we move on to llama3 instead?

Reply:
Great idea!